pkg/operator/baremetal_test.go

```diff
@@ -206,13 +206,13 @@ func TestGetMetal3DeploymentConfig(t *testing.T) {
 		t.Logf("Unstructed Config:  %+v", u)
 		t.Errorf("Failed to get Baremetal Provisioning Config from CR %s", testConfigResource)
 	}
-	actualCacheURL := getMetal3DeploymentConfig("CACHEURL", baremetalConfig)
+	actualCacheURL := getMetal3DeploymentConfig("CACHEURL", *baremetalConfig)
```

Comment:
Is it a big deal to change the function to take a pointer? Dereferencing the pointer in every call is a bit expensive, since you'll cause an entire copy of the config to be passed each time, instead of a copy of the address.

Reply:
> Is it a big deal to change the function to take a pointer? Dereferencing the pointer in every call is a bit expensive, since you'll cause an entire copy of the config to be passed each time, instead of a copy of the address.

This is the unit test code.